server/src/com/thoughtworks/go/config/GoConfigDao.java

```diff
@@ -624,7 +624,42 @@ public CruiseConfig update(CruiseConfig cruiseConfig) throws Exception {
         }
     }
 
+    public static class UpdateEnvironmentCommand implements UpdateConfigCommand {
+
+        private final String environmentToUpdate;
+        private final EnvironmentConfig newEnvironmentConfig;
+
+        public UpdateEnvironmentCommand(String environmentToUpdate, EnvironmentConfig newEnvironmentConfig) {
```

Comment:
Isn't `environmentToUpdate` supposed to be the same as `newEnvironmentConfig.name()`?